### ISSUE TITLE
Use color picker without clicking any button

### ISSUE THREAD

**Issue by andretse77** [KroneCorylus/ngx-colors#120]
The way ngx-colors is built, first you have to click on the color (or icon) to show up the colors palette and another button which will then lead you to the color picker. 
Is there any way to show directly the color picker without the need to click any button?

**Comment by KroneCorylus** [comment]
i dont think this is posible right now, it will on v4

**Comment by KroneCorylus** [comment]
duplicated of #105, there @poirierlouis provided a workaround.

**Comment by andretse77** [comment]
> duplicated of #105, there @poirierlouis provided a workaround.

It's not the same issue. The way it is, I still have to click one button in order to access the color picker. What I wanted was to display the color picker without having to click any button as it becomes less intuitive for users to have to click on a button to access a color picker.
![image](https://github.com/user-attachments/assets/d11255d4-8f23-4746-8290-7f6d2f79c58d)
![image](https://github.com/user-attachments/assets/c79c338a-f48d-4b64-aebb-ea9e6ed724a1)

**Comment by KroneCorylus** [comment]
Hey @andretse77,

Just to make sure I'm on the same page: you're looking for the panel to open directly on the color picker (sliders) when the trigger is clicked, skipping clicking on this:
![image](https://github.com/user-attachments/assets/7cb2ea5d-1337-4549-971a-13779befe046)
im correct ?

If that's the case, the workaround mentioned in #105 can still help. It involves forcing the panel so that inmediatly switches to menu = 3 (sliders) when is opened

**Comment by andretse77** [comment]
Yes, I want to skip the click of that button, but I also want to skip the use of this button
![image](https://github.com/user-attachments/assets/0e05c0a8-d55b-49ae-a44c-59206000d863)

The goal is to access the color picker without having to click on any buttons. 
#105 has helped me to skip on 1 button, but I still need to click on the "color button" to get to the color picker. I've tried working from #105, but nothing has worked. Have I explained myself clear, or am I not explaining myself very well?

**Comment by KroneCorylus** [comment]
Yeah, now I understand.
 unfortunately the library dont expose that component as a standalone. it will on 4.0 but it still needs some time to be ready to release. I recommend using other library like https://github.com/zefoy/ngx-color-picker that already have that functionality.

**Comment by KroneCorylus** [comment]
https://ngx-colors.web.app/examples#palette-only

hiding the palette using display options lets you access the color picker sliders directly. To not use the trigger and have the panel directly as a standalone component, there is already another issue (#94), so im going to close this one.
